api/proto/node/node.go

```diff
@@ -137,6 +139,21 @@ func ConstructTransactionListMessageAccount(transactions types.Transactions) []b
 	return byteBuffer.Bytes()
 }
 
+// ConstructStakingTransactionListMessageAccount constructs serialized staking transactions in account model
+func ConstructStakingTransactionListMessageAccount(transactions types2.StakingTransactions) []byte {
+	byteBuffer := bytes.NewBuffer([]byte{byte(proto.Node)})
```

Comment:
maybe `[]byte{byte(proto.Node), byte(Staking), byte(Send)})`?

Reply:
Ack. Let's do it later.